resources/core/charts/service-catalog-addons/charts/brokers-ui/templates/brokers-microfrontend.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: "ui.kyma-project.io/v1alpha1"
```

Comment:
brokers-ui/Chart.yaml is not touched, should be?, I see you upgraded versions for catalog-ui and instances-iu

Reply:
Brokers UI chart does not have to be touched as the view itself did not have to be touched.